loclist.kak

```diff
@@ -0,0 +1,147 @@
+## Location List
+
+declare-option -hidden str loclist_buffer
```

Comment:
I tend to stick to the rule of "no abbreviations in global names".
Some ideas intead of `loclist`
- `locations` - simple, somewhat nondescript
- `jump-list` - seems nice; kind of taken (I think Kakoune's `jumplist` should be renamed to the more fitting `jumpstack`)
- `quick-fix` - the Vim term
- `marks` - means something else in Vim
- `goto`/`goto-list` - meh
No rush to decide, can let it simmer.

Reply:
locations is the best one there i think. jump-list would be better, but as you're saying its taken. quick-fix is just confusing, marks is too general, goto-list meh.

locations seems like the best way to move forward

Reply:
I think it should be ready now. We can look at integrating `locations-write` later